bin/ycsb

```diff
@@ -87,7 +87,7 @@ DATABASES = {
     "maprdb"       : "site.ycsb.db.mapr.MapRDBClient",
     "maprjsondb"   : "site.ycsb.db.mapr.MapRJSONDBClient",
     "mongodb"      : "site.ycsb.db.MongoDbClient",
-    "mongodb-async": "site.ycsb.db.AsyncMongoDbClient",
```

Comment:
[RaviTella](https://github.com/RaviTella) lets get this back

Reply:
done!